### ISSUE TITLE
Key bindings aren't given for aliases

### ISSUE THREAD

**Issue by nickdrozd** [Wilfred/helpful#143]
I have `qrr` bound as an alias for `query-replace-regexp`. Key bindings are shown for the latter, but not the former.

**Comment by Wilfred** [comment]
Fixed, thanks.

**Comment by nickdrozd** [comment]
Hmm, this doesn't seem to be working for me, or maybe I wasn't clear about the issue.

For `query-replace-regexp`, it gives
```
Key Bindings
esc-map C-%
global-map C-M-%
```

For `qrr`, it gives
```
Key Bindings
This command is not in any keymaps.
```

I would expect the `query-replace-regexp` key bindings to be given for `qrr`, or else maybe a note that says "see `query-replace-regexp` or something like that.

**Comment by nickdrozd** [comment]
Oh, I get it. If you look very closely, I think you'll see where the problem is :)

```
(defun helpful--keymaps-containing-aliases (command-sym)
  "Return a list of pairs mapping keymap symbols to the
keybindings for COMMAND-SYM in each keymap.

Includes keybindings for aliases, unlike
`helpful--keymaps-containing'."
  (let* ((aliases (helpful--aliases 'helpful--dummy-command t))
         (syms (cons command-sym aliases))
         (syms-keymaps (-map #'helpful--keymaps-containing syms)))
    (-reduce #'helpful--merge-alists syms-keymaps)))
```

**Comment by Wilfred** [comment]
Woops! That's a risk when writing code incrementally :)

Fixed in c4178fa, could you try again?
